docs/essentials/sensors.md

```diff
@@ -0,0 +1,540 @@
+---
+title: Accessing device sensors overview
+description: "Learn how to use and monitor sensors provided by your device, with .NET MAUI. You can monitor the following sensors: accelerometer, barometer, compass, shake, gyroscope, magnetometer, orientation."
+ms.topic: overview
+ms.date: 08/09/2021
+ms.custom: template-overview
+show_latex: true
+---
+
+# Accessing device sensors
+
+Devices have all sorts of sensors available to you. Some sensors can detect movement, others changes in the environment, such as light. Monitoring and reacting to these sensors makes your app dynamic in adapting to how the device is being used. You can also respond to changes in the sensors and alert the user. This article gives you a brief overview of the common sensors supported by .NET Multi-User Application (.NET MAUI).
+
+## Sensor speed
+
+Sensor speed sets the speed in which a sensor will return data to your app. When you start a sensor, you provide the desired sensor speed. However, monitoring too many sensors at once may affect the rate sensor data is returned to your app.
+
+- `Fastest`\
```

Comment:
What is `Fastest` about? I can see it's a type. From the text I know that I have to provide the sensor speed when starting the sensor. Presumably these are all enumeration members? I think it needs re-arranging to something like:

----

Sensor speed sets the speed in which a sensor will return data to your app. When you start a sensor, you provide the desired sensor speed with the `SensorSpeed` enumeration, which provides the following members:

<enumeration members go here>

However, monitoring too many sensors at once may affect the rate sensor data is returned to your app.

----

You could even make the final sentence a note or warning?